dart/test/sentry_tracer_test.dart

```diff
@@ -386,6 +386,28 @@ void main() {
       expect(sut.endTimestamp, endTimestamp);
     });
 
+    test('end trimmed to latest child end timestamp', () async {
+      final sut = fixture.getSut(trimEnd: true);
+      final rootEndInitial = getUtcDateTime();
+      final childEnd1 = rootEndInitial;
+      final childEnd2 = rootEndInitial.add(Duration(seconds: 1));
+      final childEnd3 = rootEndInitial;
```

Comment:
`l`: I would stick to one naming pattern. Either name everything A, B, C or 1, 2, 3.
```suggestion
      final childAEnd = rootEndInitial;
      final childBEnd = rootEndInitial.add(Duration(seconds: 1));
      final childCEnd = rootEndInitial;
```